about.html

```diff
@@ -86,7 +112,7 @@ <h2>
         </p>
       </div>
       <figure class="logo-img">
-        <img src="./assets/img/logo.PNG" alt="" />
+        <img class="img-thumbnail" src="./assets/img/logo.PNG" alt="" />
```

Comment:
- [ ] as mentioned by the previous reviewer kindly reduce the size of the logo to match the template

 

![image](https://user-images.githubusercontent.com/89282221/190502744-85b40957-24d5-41c1-a79c-86873a33ea24.png)